src/v2/view-builder/views/custom-app/ChallengeCustomAppPushView.js

```diff
@@ -0,0 +1,84 @@
+import { _, loc, createButton, View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import { BaseForm } from '../../internals';
+import polling from '../shared/polling';
+import { WARNING_TIMEOUT } from '../../utils/Constants';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+
+const warningTemplate = View.extend({
+  className: 'okta-form-infobox-warning infobox infobox-warning',
+  template: hbs`
+    <span class="icon warning-16"></span>
+    <p>{{warning}}</p>
+  `
+});
+
+const Body = BaseForm.extend(Object.assign(
+  {
+    noButtonBar: true,
+
+    className: 'custom-app-push-challenge',
+
+    title() {
+      return loc('oie.verify.custom_app.title', 'login', [this.options.appState.getAuthenticatorDisplayName()]);
+    },
+
+    initialize() {
+      BaseForm.prototype.initialize.apply(this, arguments);
+      this.listenTo(this.model, 'error', this.stopPoll);
+      this.addView();
+    },
+
+    addView() {
+      this.add(createButton({
+        className: 'button button-wide button-primary send-push link-button-disabled',
+        title: loc('oie.custom_app.push.sent', 'login'),
```

Comment:
Doesn't this need to be a function to ensure it get localized?

Reply:
@jmelberg-okta the loc function should take care of it right. anyway I double checked and it looks fine . ( I just added a string to french prop file to check manually'

![Screen Shot 2021-08-17 at 8 53 55 AM](https://user-images.githubusercontent.com/17267130/129759637-bed4c430-e4e8-4116-96f8-306e92056227.png)

Reply:
Thanks for checking! Was this from an uncached state (i.e. `osw.languages` local storage object didn't exist)?

My concern was mostly calling out that there are some cases when the `title` field needs to return a function (our top-level views do this), due to the async nature of our language loading.